src/types/onyx/IOU.ts

```diff
@@ -1,10 +1,20 @@
+import {Icon} from './OnyxCommon';
+
 type Participant = {
     accountID: number;
-    login?: string;
+    login: string | undefined;
```

Comment:
Why this change?

Reply:
it was needed for this 
https://github.com/Expensify/App/blob/8bef4bbd123fb6aa8d36e91c3012677208670793/src/libs/OptionsListUtils.js#L1224-L1231

when login was optional it was complaining thats why I changed this to be required but can be undefined